Podfile.lock

```diff
@@ -266,4 +228,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: d08615d60b668dfbf99e70a4e11e3869879ca70d
 
-COCOAPODS: 1.5.3
+COCOAPODS: 1.3.1
```

Comment:
You're using an older version of cocoapods. That might work fine, but let's keep the change out of the repo.